unattended_installer/install_functions/installMain.sh

```diff
@@ -207,7 +207,7 @@ function main() {
     fi
 
     common_logger "Starting Wazuh installation assistant. Wazuh version: ${wazuh_version}"
-    common_logger "Verbose logging redirected to ${logfile}"
+    common_logger "Verbose logging will be redirected to ${logfile}"
```

Comment:
I think it's better not to change that according to content team advice: do not promise, just do it.

Reply:
Done: https://github.com/wazuh/wazuh-packages/pull/1440/commits/fe9c6f8f6f7d0532bcbf75230fd5ba819a092e48